python/pipelines/pipeline_ops.py

```diff
@@ -222,7 +258,7 @@ def compile_automl_tabular_pipeline(
         parameter_values,
     ) = automl_tabular_utils.get_automl_tabular_pipeline_and_parameters(**pipeline_parameters)
 
-    with open(pathlib.Path(__file__).parent.resolve().joinpath('automl_tabular_pl_v2.yaml'), 'r') as file:
+    with open(pathlib.Path(__file__).parent.resolve().joinpath('automl_tabular_pl_v3.yaml'), 'r') as file:
```

Comment:
This yaml file of the pipelines eventually it will be returned from the library using  automl_tabular_utils.get_automl_tabular_pipeline_and_parameters. Did we check with the team that it is not yet included in the library? (initial issue the library only returned .json and not .yaml. They were going to fix it in future release. we need un update from the team)